src/ctia/stores/es/query.clj

```diff
@@ -2,6 +2,34 @@
   (:require [clj-momo.lib.es.query :as q]
             [clojure.string :as str]))
 
+(def public-tlps
+  ["white" "green"])
+
+(defn find-restriction-query-part
+  [{:keys [login groups]}]
+  ;; TODO do we really want to discard case on that?
+  (let [login (str/lower-case login)
+        groups (map str/lower-case groups)]
```

Comment:
I think we should take care that the logins and groups  cannot contains any special chars that would have a special meaning in Lucene queries.

The question is, where should we protect them, here or in the auth? Or maybe in both places.

Reply:
it looks like the `terms` query part expects an exact match 
https://www.elastic.co/guide/en/elasticsearch/reference/5.4/query-dsl-terms-query.html

you need to use `wildcard` to perform wildcard searches:
https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-wildcard-query.html